pkgs/top-level/aliases.nix

```diff
@@ -850,7 +850,6 @@ mapAliases ({
   navit = throw "navit has been removed from nixpkgs, due to being unmaintained"; # Added 2021-06-07
   ncat = throw "'ncat' has been renamed to/replaced by 'nmap'"; # Converted to throw 2022-02-22
   neap = throw "neap was removed from nixpkgs, as it relies on python2"; # Added 2022-01-12
-  netcat-openbsd = throw "'netcat-openbsd' has been renamed to/replaced by 'libressl.nc'"; # Converted to throw 2022-02-22
```

Comment:
what's wrong with `libressl.nc`? why reintroduce this?

Reply:
it doesn't have -q

Reply:
So why was it removed, in the first place?

Reply:
https://github.com/NixOS/nixpkgs/pull/39634